tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -27,16 +27,16 @@ object Discover {
   def GC(): GC =
     getenv("SCALANATIVE_GC").map(build.GC(_)).getOrElse(build.GC.default)
 
-  /** Find the newest compatible clang binary. */
+  /** Use the clang binary on the path or via CLANG_PATH env var. */
   def clang(): Path = {
-    val path = discover("clang", clangVersions)
```

Comment:
So should I just call it like this and remove all the internal code to simplify things?
```scala
val path = discover("clang", "CLANG_PATH")
// and
val path = discover("clang++", "CLANGPP_PATH")
```